### PR TITLE
PR into main with fileTest

### PR DESCRIPTION
**Pull request by jaxels10**
Added condition for several storage classes
Removed comments from values file

### CODE REVIEW

File: chart/templates/object-storageclass.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: StorageClass
+metadata:
+   name: rook-ceph-bucket
+# Change "rook-ceph" provisioner prefix to match the operator namespace if needed
+provisioner: rook-ceph.ceph.rook.io/bucket
+reclaimPolicy: Retain
+parameters:
+  objectStoreName: my-store
+  objectStoreNamespace: rook-ceph
+  region: us-east-1
```
Comment (skovbakke) [inline chart/templates/object-storageclass.yaml:11]: why do we set this region?

File: chart/templates/service-monitor.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    matchNames:
+      - rook-ceph
+  selector:
+    matchLabels:
+      app: rook-ceph-mgr
+      rook_cluster: rook-ceph
+  endpoints:
+  - port: http-metrics
+    path: /metrics
+    interval: 5s
```
Comment (skovbakke) [inline chart/templates/service-monitor.yaml:18]: Is it possible to modify this such that the service monitor is only deployed if prometheus is enabled?

Comment (skovbakke) [reply chart/templates/service-monitor.yaml:18]: In the values file there is `cluster.prometheusEnabled`

**Review by skovbakke — COMMENTED** [review]
It looks good. I only have a few remarks

File: chart/templates/service-monitor.yaml (hunk 2 above)
Comment (jaxels10) [reply chart/templates/service-monitor.yaml:18]: I actually believe we should move this service monitor to the prometheus umbrella chart, so I will remove it from this PR

File: chart/templates/object-storageclass.yaml (hunk 1 above)
Comment (jaxels10) [reply chart/templates/object-storageclass.yaml]: I have removed this

**Review by skovbakke — APPROVED** [review]
Nice

**Pull request merged**